packages/cli/src/commands/config/schema.ts

```diff
@@ -28,19 +28,23 @@ export default async (opts: OptionValues) => {
     mockEnv: true,
   });
 
-  const merged = mergeConfigSchemas(
-    (schema.serialize().schemas as JsonObject[]).map(
-      _ => _.value as JSONSchema,
-    ),
-  );
-
-  merged.title = 'Application Configuration Schema';
-  merged.description =
-    'This is the schema describing the structure of the app-config.yaml configuration file.';
+  let configSchema: JsonObject | JSONSchema;
+  if (opts.merged) {
```

Comment:
```suggestion
  if (opts.merge) {
```

This might read a bit nicer with `--no-merge and --merge`

Reply:
Alright lets go for it